src/js/components/FormField/FormField.js

```diff
@@ -365,13 +302,9 @@ const FormField = forwardRef(
         margin={abut ? abutMargin : margin || { ...formFieldTheme.margin }}
         {...outerProps}
         style={outerStyle}
-        onFocus={event => {
-          setFocus(true);
-          if (onFocus) onFocus(event);
-        }}
+        onFocus={() => setFocus(true)}
```

Comment:
Where else would the caller `onFocus` would be called?

Reply:
restored